_migrations/getting-started-data-migration.md

```diff
@@ -0,0 +1,329 @@
+
+#### `ReindexFromSnapshot` parameters
+
+If you're creating a snapshot using migration tooling, these parameters are auto-configured. If you're using an existing snapshot, modify the  `reindexFromSnapshotExtraArgs` setting with the following values:
+
+    ```bash
+    --s3-repo-uri s3://<bucket-name>/<repo> --s3-region <region> --snapshot-name <name>
+    ```
+
+You will also need to give access to the `migrationconsole` and `reindexFromSnapshot` taskRole permissions to the EC2 bucket. 
+
+---
+
+## Step 5: Deploying the migration assistant
+
+To deploy the migration assistant to EC2, use the following steps:
+
+1. Bootstrap the account:
+   
+    ```bash
+    cdk bootstrap --c contextId=migration-assistant --require-approval never --concurrency 5
+    ```
+2. Deploy the stacks when `cdk.context.json` is fully configured:
+   
+    ```bash
+    cdk deploy "*" --c contextId=migration-assistant --require-approval never --concurrency 3
+    ```
+
+These commands deploy the following stacks:
+
+* Migration assistant network stack
+* RFS stack
+* Migration console stack
+
+---
+
+## Step 6: Accessing the Migration Console
+
+Run the following command to access the migration console:
+
+```bash
+./accessContainer.sh migration-console dev <region>
+```
+
+
+`accessContainer.sh` is located in `/opensearch-migrations/deployment/cdk/opensearch-service-migration/` on the bootstrap instance. To learn more, see [Accessing the migration console]:
+`{: .note}
+
+---
+
+## Step 7: Checking the connection to the source and target clusters
+
+To verify the connection to the clusters, run:
+
+```bash
+console clusters connection-check
+```
+
+You should receive the following output:
+
+```
+* **Source Cluster:** Successfully connected!
+* **Target Cluster:** Successfully connected!
+```
+
+To learn more about migration console commands, see [Migration commands].
+
+---
+
+## Step 8: Snapshot creation
+
+Run the following to initiate creating a snapshot from the source cluster:
+
+```bash
+console snapshot create [...]
+```
+
+To check on the progress of the snapshot creation, use:
+
+```bash
+console snapshot status [...]
+```
+
+To learn more details about the snapshot, use:
+
+```bash
+console snapshot status --deep-check [...]
+```
+
+Wait for the snapshot to complete before moving to step 9.
+
+To learn more about snapshot creation, see [Snapshot Creation]:
+
+---
+
+## Step 9 - Metadata Migration
+
+Run the following command to migrate metadata:
+
+```bash
+console metadata migrate [...]
+```
+
+To learn more see [Metadata migration]
+
+---
+
+## Step 10: RFS Document Migration
+
+You can now use RFS to migrate documents from your original cluster:
+
+1. To start the migration from RFS, start a `backfill` using the following command:
+
+    ```bash
+    console backfill start
+    ```
+
+2. If you need more workers in order to accommodate the number of documents in your cluster, use the following command:
```

Comment:
So the first command performs a faster migration and this command uses an allocated number of workers to create a more stable one?

Reply:
On second thought, reread your suggestion and this reads more like an optional step than a required one. What would the user use for the more "stable" option?

Reply:
I'd say we relabel this as optional step.  Its highly variable based on the user cluster if it will 1) have any positive impact 2) have any negative impact.

While there are definitely ways this can be tweaked those wouldn't be called out in this quick start guide and would be covered in by the `Backfill Execution` link below this list.